### ISSUE TITLE
Upgrade from C117 -> C118 on macOS x64 via 1.60.x - Nightly (Checking C118)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#33231]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by LaurenWags** [comment]
Verification in progress with

```
Brave | 1.60.47 Chromium: 118.0.5993.21 (Official Build) nightly (x86_64)
-- | --
Revision | 8c2edac2b59451745c32ee03f6aca51abfe4d222
OS | macOS Version 13.6 (Build 22G120)
```

**Comment by LaurenWags** [comment]
Verified with

```
Brave | 1.60.47 Chromium: 118.0.5993.21 (Official Build) nightly (x86_64)
-- | --
Revision | 8c2edac2b59451745c32ee03f6aca51abfe4d222
OS | macOS Version 13.6 (Build 22G120)
```

<details>
<summary>C118 --> C118 on-demand upgrade check - InProgress</summary>

1. Install C118 PR build (x64 builds available via https://bravesoftware.slack.com/archives/CU5UXM3EC/p1695467144813629)
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit brave://settings/help but close this tab prior to closing the browser)


4. Close
5. Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates.brave.com/sparkle/Brave-Browser/test/appcast.xml` (note, changed URL)
6. Navigate to brave://settings/help
7. See that an update is found

```

```


8. Confirm you are prompted to "Relaunch" to get update


9. If you click "Relaunch" button you will restart the browser, but this time without the flags specified in the terminal, as such another update will be found (since there are newer C116 versions of Nightly available on non-staging sparkle). If you close the browser manually and relaunch via the terminal using the same command as in step 5 (not using "Relaunch" button), you will not be updated again.

Using "Relaunch" button | Manual close/reopen
--- | ---


</details>
